hibp.js

```diff
@@ -145,7 +145,7 @@ const HIBP = {
                 !breach.IsSpamList &&
                 !breach.IsFabricated &&
                 breach.IsVerified &&
-                !(breach.Domain === "" && breach.Name !== "PDL")
+                !(breach.Domain === "")
```

Comment:
nit: Should this just be `breach.Domain !== ""` instead of the extra step of `===` and then negating it with `!`?